### PR TITLE
Translate "A Tour of Sage" into Greek

### PR DESCRIPTION
**Pull request by gmou3**
### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by saraedum** [comment]
@gmou3 I could in principle review this. I am not a native speaker but I live in Athens so I am not completely uncomfortable reading this.

Do you have another reviewer in mind already for this?

**Comment by gmou3** [comment]
No, I don't. This is the first time I translate something for sage.

**Comment by saraedum** [comment]
Are you a native speaker? @gmou3

**Comment by gmou3** [comment]
Yes.

File: src/doc/el/a_tour_of_sage/index.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
+.. _a-tour-of-sage:
+
```
Comment (saraedum) [inline src/doc/el/a_tour_of_sage/index.rst]:
```suggestion
```

 I don't think this marker should be here. Not sure if it conflicts with marker in the English version. But the other translations don't seem to have it and it's not referenced anywhere.

File: src/doc/el/a_tour_of_sage/index.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+με τους οποίους ο αριθμός 100 εκατομμύρια μπορεί να γραφεί ως άθροισμα θετικών
+ακεραίων.
+
+::
+
+    sage: z = Partitions(10^8).cardinality()  # περίπου .1 δευτερόλεπτα
+    sage: z
+    1760517045946249141360373894679135204009...
+
+Το Sage είναι το πιό προηγμένο λογισμικό ανοιχτού κώδικα για μαθηματικά στον
```
Comment (saraedum) [inline src/doc/el/a_tour_of_sage/index.rst]:
```suggestion
Το Sage είναι το πιο προηγμένο λογισμικό ανοιχτού κώδικα για μαθηματικά στον
```
Should this really be πιό?

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
@gmou3 sorry, took a bit longer to find the time to browse over this.
This looks great to me (but the vocabulary certainly exceeds my Greek…)
Maybe you can have a look at the two comments, and then I am happy to set this to positively reviewed.

File: src/doc/el/a_tour_of_sage/index.rst (hunk 2 above)
Comment (gmou3) [reply src/doc/el/a_tour_of_sage/index.rst]: Typically it wouldn't be accented as you suggest. I happen to be a bit more liberal with my punctuation marks, based on how I read.

**Comment by gmou3** [comment]
Thank you for your time. If you think it is desirable/useful I can also translate the tutorial.

**Pull request merged**